### ISSUE TITLE
Refreshing five times before logging in takes quite a long time

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1486]
Find and fix the bug that causes the "Slowly Connecting..." message from appearing after five refreshes before logging in.

Now that Brave and Chrome have fixed their regression (#1485), I opened a window in Brave, went to https://craftpoker.com, refreshed a bunch of times and then tried to log in. It failed, with me getting the "Slowly Connecting..." message and no actual login for minutes. I used a binary search to find that if I refresh five times and then try to log in, that happens, but if I refresh four times and try to log in, it doesn't.

This appears to be the same issue as White screen when rapidly refreshing demo windows (#1390), but this doesn't involve demo windows and it's super curious to me that four refreshes and the logging in works instantly but five refreshes and then logging in takes minutes. I do, however, have a bunch of other windows and tabs open and there's a chance that when I close them there will be a difference.

**Comment by ctm** [comment]
I closed all my windows and tabs except GMail and now I can't reproduce the behavior. I opened craftpoker.com, refreshed 50 times and then logged in without incident. FWIW, many of the tabs I had open were Strava or were BF 200 2024 photo collections. I haven't looked to see if either of those use WebSockets. I doubt they do, but I'll try opening up one of each and poke around in the Network tab. After that, perhaps I'll try opening a bunch and trying again.

It looks like this isn't an mb2 bug, but a Chromium (or much less likely, Brave) bug.

**Comment by ctm** [comment]
FWIW, the Strava window I opened didn't use websockets or wasm, but the BF 200 photo page did use a websocket. So, I opened all eleven links on the [summary page](https://destination200photos.com/f328205857) and tried to reproduce this problem, but without success. Even with those eleven tabs open and sixteen refreshes of craftpoker.com, I was able to log in quickly.

I'm stripping `high priority` and even `bug` from this and adding `won't fix`.
